examples/dynamic-bridge-app/linux/include/data-model/Attribute.h

```diff
@@ -52,6 +54,22 @@ struct Attribute : public AttributeInterface
         return chip::app::DataModel::Encode(aPath, aEncoder, mData);
     }
 
+    template <typename T = Type, std::enable_if_t<chip::app::DataModel::IsList<std::decay_t<T>>::value, bool> = true>
+    CHIP_ERROR ReadValue(const chip::app::ConcreteReadAttributePath & aPath, chip::TLV::TLVWriter & writer, Type & value)
+    {
+        return CHIP_ERROR_NOT_IMPLEMENTED;
```

Comment:
Add a comment of sideffects here: does this mean that all lists are not yet supported by bridges? Should we add some log message when we get this? SHould there be a todo to implement it?

Reply:
Added support for lists in bridges. See [ec0b498](https://github.com/project-chip/connectedhomeip/pull/23840/commits/ec0b498772e355a470d0c9bb2157696d58b91ca8)